src/game-engine/application-states/PlayerA/ReadyToChooseAPlay.ts

```diff
@@ -0,0 +1,11 @@
+import BasePlayerA from './BaseState';
+import Message from '../../Message';
+
+export default class ReadyToChooseAPlay extends BasePlayerA {
+  opponentMessage: Message;
+  constructor({ channel, stake, balances, adjudicator, opponentMessage }) {
```

Comment:
Might be nice to change all these constructors to take individual arguments with types, instead of the named arguments that seem not to work so well with typescript.

Reply:
also upcoming on the chopping block. a lot of constructors have already been changed, I think? But I never got around to these yet.